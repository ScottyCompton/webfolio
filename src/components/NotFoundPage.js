import React from 'react';
import {Link} from 'react-router-dom';


const NotFoundPage = () => (
    <div className="container">
        <div className="row">
            <div className="col-xs-12 col-sm-12 col-md-12 col-lg-12">
            <div><h1>Error 404</h1></div>
                <p>&nbsp;</p>
                <h1 className="text-center">Whatever you're looking for,</h1>
                <div className="text-center">
                    <img src="https://media.giphy.com/media/Uy95VBz69cGQg/giphy.gif" />
                </div>
                <div className="text-center">
                    <h1>It's not here.</h1>
                </div>
                <div className="text-center">
                    <p>&nbsp;</p>
                    <Link to="/" class="btn btn-warning">Leave this place</Link>
                    <p>&nbsp;</p>
                    <p>&nbsp;</p>
                </div>
            </div>
        </div>
    </div>
);


export default NotFoundPage;