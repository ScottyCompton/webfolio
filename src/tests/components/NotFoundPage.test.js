import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';

test('Should render single expense list item', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();

});