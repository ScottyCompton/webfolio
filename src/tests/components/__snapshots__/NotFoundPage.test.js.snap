// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render single expense list item 1`] = `
<div>
  404! - 
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
